includes/class-wc-google-analytics.php

```diff
@@ -722,6 +722,13 @@ public function async_script_loader_tags( $tag, $handle, $src ) {
 			return $tag;
 		}
 
-		return str_replace( '<script src', '<script async src', $tag );
+		// Check if the script has the async attribute already. If so, don't add it again.
+		$has_async_tag = preg_match( '/^.*async.*$/', $tag );
```

Comment:
Two `async` attributes still occur as this regex pattern cannot match correctly. Maybe consider matching it with a pattern like `/\basync\b/`?